docs/fuse.md

```diff
@@ -26,6 +28,12 @@ go get github.com/jbenet/go-fuse-version/fuse-version
 
 If you run into any problems installing FUSE or mounting IPFS, hop on IRC and speak with us, or if you figure something new out, please add to this document!
 
+You may also have to add your username to the `operator` group:
```

Comment:
I'm not sure. I had not tested it without it. `/dev/osxfuse*` all have `root:operator`. Might still work without it, which is why I said "might".